src/textual/devtools/client.py

```diff
@@ -3,19 +3,34 @@
 import asyncio
 import inspect
 import json
-import msgpack
+
 import pickle
 from time import time
 from asyncio import Queue, Task, QueueFull
 from io import StringIO
 from typing import Type, Any, NamedTuple
 
-import aiohttp
-from aiohttp import ClientResponseError, ClientConnectorError, ClientWebSocketResponse
 from rich.console import Console
 from rich.segment import Segment
 
 
+class DevtoolsDependenciesMissingError(Exception):
+    """Raise when the required devtools dependencies are not installed in the environment"""
+
+
+try:
+    import aiohttp
+    from aiohttp import (
+        ClientResponseError,
+        ClientConnectorError,
+        ClientWebSocketResponse,
+    )
+    import msgpack
+except ImportError:
+    raise DevtoolsDependenciesMissingError(
+        "Textual Devtools requires installation of the 'dev' extra dependencies. "
```

Comment:
Can you add the command required to install the dev extras? We should always assume novice devs.

Reply:
A link to the docs would work. Maybe add a TODO then...